File: src/app/Models/compte.ts
export interface Compte {
    id: number;
    nomUtilisateur: string;
    motDePasse: string;
    dateDerniereConnexion: Date;
}

export interface CompteConnexionRequete {
    nomUtilisateur: string;
    motDePasse: string;
}

export interface CompteCreationRequete {
    nomUtilisateur: string;
    motDePasse: string;
}

export interface CompteConnexionReponse {
    jeton: string;
    nomUtilisateur: string;
}

File: MemosWebApplicationCore/Entites/Compte.cs


namespace MemosWebApplicationCore.Entites;

public class Compte: BaseEntity
{
    public string NomUtilisateur { get; set; } = string.Empty;

    public string MotDePasse { get; set; } = string.Empty;

    public DateTime DateCreation { get; set; }

    public DateTime? DateDerniereConnexion { get; set; }

    public virtual ICollection<Memo> Memos { get; set; } = new List<Memo>();
}
